modules/format_strawberryfield_views/src/Plugin/views/filter/AdvancedSearchApiFulltext.php

```diff
@@ -578,15 +578,18 @@ public function buildExposedForm(&$form, FormStateInterface $form_state) {
         '#default_value' => $form_state->getValue($searched_fields_identifier),
       ];
 
-      if (isset($form[$this->options['id'] . '_wrapper'])) {
-        $form[$this->options['id'] . '_wrapper'][$searched_fields_identifier]
-          = $newelements[$searched_fields_identifier];
+      if (empty($form[$this->options['id'] . '_wrapper'])) {
+        $form[$this->options['id'] . '_wrapper'] =
+          [
+            "#type" => "fieldset",
```

Comment:
@patdunlavey this here should better come from the original Method, in case they decide to change how the wrapper is build (at the extended Filter class)

protected function buildValueWrapper(&$form, $wrapper_identifier) {
    // If both the field and the operator are exposed, this will end up being
    // called twice. We don't want to wipe out what's already there, so if it
    // exists already, do nothing.
    if (!isset($form[$wrapper_identifier])) {
      $form[$wrapper_identifier] = [
        '#type' => 'fieldset',
      ];

      $exposed_info = $this->exposedInfo();
      if (!empty($exposed_info['label'])) {
        $form[$wrapper_identifier]['#title'] = $exposed_info['label'];
      }
      if (!empty($exposed_info['description'])) {
        $form[$wrapper_identifier]['#description'] = $exposed_info['description'];
      }
    }
  }


So basically
$wrapper = $form[$this->options['id'] . '_wrapper'];
$this->buildValueWrapper($form, $wrapper);

That also makes the code slimmer. What do you think?

Reply:
Does this fix it?